docs/spec_draft.md

```diff
@@ -442,6 +443,48 @@ produces an implementation-defined value.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.exp
+
+`stablehlo.exp(operand) -> result`
+
+### Semantics
+
+Performs element-wise exponential operation on `operand` tensor and produces a
+`result` tensor. For floating-point element types, it implements the `exp`
+operation from the IEEE-754 specification. For complex element types, it
+computes a complex form of exponential using Euler's formula, with corner cases
```

Comment:
Use a simplified version of this consistent with other ops, e.g. "For complex element types, it computes a complex exponential, with corner cases TBD"?

Reply:
As the Euler's formal is taking care of the imaginary part only as
![image](https://user-images.githubusercontent.com/5316885/190506655-a833339a-3991-48c4-8ef4-c9cc3b8e6469.png)

How about we explicit write our formula as
$e^{a+ix} = e^a . e^{ix}$ where $e^{ix}$ is the complex exponential and the multiplication is the `multiplication` operation from IEEE-754 standard.

Reply:
After looking into https://en.cppreference.com/w/cpp/numeric/complex/exp, I realized that "complex exponentiation function" is not can mean $e^{a+ix}$ as well. 

Let me take back my comment.